constants.ts

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
I prefer we undo `constants.ts` and keep them close to the where they are called, this makes it easier to review/maintain. This codebase only has 3 key files anyway.